src/app/CommandResponseHelper.h

```diff
@@ -40,9 +41,22 @@ class CommandResponseHelper
         return CHIP_NO_ERROR;
     };
 
+    // WARNING: Do not use for basic response-payload-free success, as SUCCESS status
```

Comment:
I am not sure I completely follow this warning. Could you expand with some examples of legal and illegal (and why) usages?